### ISSUE TITLE
BIMserver not recognising plugins (WebGL-threeJS)

### ISSUE THREAD

**Issue by connoralexander** [opensourceBIM/BIMserver#252]
Hey, so when I drop .jar plugin files into the folder inside the BIMserver directory they are not being recognised in BIMvie.ws as there. Ie. When I go to the plugins dropdown it is not there, it is not listed under serializers or viewers or anything.

Specifically I am trying to load in the viewer https://github.com/opensourceBIM/WebGL-threeJS

I opened an issue on there but it may be the BIMserver code which needs editing? Any help would be great as I really need the server side part working so I can start manipulating the client side and playing with exported JSON.

**Comment by hlg** [comment]
The plugin is shown after restarting the server. However, I realized the following related issues:
* If `plugin.xml` has `enabled` set to `false`, restarting BIMserver does not have an effect on what is shown in BIMvie.ws (depending on the previous state, it is either not shown at all or it is still there and enabled). In theory I think this setting should allow to distribute a plugin which is disabled by default and needs to be explicitely enabled.
* If a plugin file is removed and BIMserver restarted, the plugin is still shown in the plugin list. Accessing the plugin details yields an error: "no plugin with class name ... found".
The latter seems like a BIMvie.ws bug to me, for the former I did not check whether it is just an issue of the UI or an issue of the plugin loading in BIMserver.
